app/js/controllers/eboard-ctrl.js

```diff
@@ -83,13 +83,27 @@ angular
 
                     // Add the person to team. This simply just adds the person to the members array
                     // for that particular team in the team scope.
-                    $scope.teams[valWithoutLead].members.push({
-                        'name': person.attributes.name,
-                        'imgUrl': person.attributes.imgUrl,
-                        'uniqId': index,
-                        'apiId': person.id,
-                        'isLead': isLead
-                    });
+                    if (person.attributes.imgUrl) {
```

Comment:
So instead of doing this if /else with lots of overlying fields (`person.attributes.name` is the same for example). 

Do this:
`//create new object with fields that will always be the same`
`// do the if person has imgURL`
`//if true set the objects.imgUrl to that`
`//if false set it to the default`
`after the if statement, push the object onto the members array`